### PR TITLE
feat(icons): add kirby icon

### PR DESCRIPTION
**Pull request by guastallaigor**
image

-----------------------
I've also changed the width of mario and ash icons that was a little bit larger than expected.

### CODE REVIEW

File: scss/pixel-arts/kirby.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
+.kirby {
```
[the remark this answers is not in the document]
Comment (guastallaigor) [reply scss/pixel-arts/kirby.scss]: Added :+1:

**Comment by guastallaigor** [comment]
**Index.html**

image

**Storybook**

image

File: scss/pixel-arts/mario.scss [hunk 2]
```diff
@@ -25,7 +25,7 @@
 
   position: relative;
   display: inline-block;
-  width: $px * 16;
+  width: $px * 13;
```
Comment (BcRikko) [inline scss/pixel-arts/mario.scss]: Here is 14 👍 
```suggestion
  width: $px * 14;
```

File: scss/pixel-arts/kirby.scss (hunk 1 above)
Comment (BcRikko) [inline scss/pixel-arts/kirby.scss]: Could you confirm it again? 🤔 
```suggestion
.nes-kirby {
```

Comment (guastallaigor) [reply scss/pixel-arts/kirby.scss]: For some reason my login don't went in so the push failed, just checked and now it should be good.

File: scss/pixel-arts/mario.scss (hunk 2 above)
Comment (guastallaigor) [reply scss/pixel-arts/mario.scss]: You're right, that's better, thank you.

**Pull request merged**